extensions/wikia/TrackingOptIn/js/trackingOptIn.js

```diff
@@ -44,9 +60,54 @@ define('wikia.trackingOptIn', [
 		window.Wikia.consentQueue.push(callback);
 	}
 
+	function geoRequiresTrackingConsent() {
+		return geoRequiresConsent;
+	}
+
+	function getPrebidGlobalVendorList() {
+		var request = new XMLHttpRequest();
+
+		log('Requesting IAB Global Vendor List', log.levels.debug, logGroup);
+
+		request.open('GET', prebidVendorListUrl, false);
+		request.send(null);
+
+		return JSON.parse(request.responseText);
+	}
+
+	function getPrebidConsentString(optIn) {
```

Comment:
Can we move building consent string to trackingOptIn module? Does it make sense? Are we going to reuse it somewhere else?

Reply:
We could, IAB consent string is a standard and other non-prebid services can (I think) get info about consent in this format, but there's nothing bad with moving it to separate module. Done.